src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let counter = 0;
+
+  const getSum = (firstNumber) => {
+    counter++;
+
+    return (secondNumber) => {
+      const shouldWeSum = counter < 3 || counter % 2 !== 0;
 
+      if (shouldWeSum) {
+        return firstNumber + secondNumber;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
Looks better, right? 
```suggestion
      const shouldSum = counter < 3 || counter % 2 !== 0;

      return shouldSum
        ? firstNumber + secondNumber
        : 'Bzzz... Error!'
      }
```